include/alpaka/core/Common.hpp

```diff
@@ -14,10 +14,13 @@
 
 //! All functions that can be used on an accelerator have to be attributed with ALPAKA_FN_ACC or ALPAKA_FN_HOST_ACC.
 //!
+//! @code{.cpp}
```

Comment:
Sry for sneaking in this change, I first commented by accident on this part and realized that only ALPAKAFN_INLINE is affected but would like to keep this change due to the correct usage of doxygen.